degirum_tools/figure_annotator.py

```diff
@@ -483,7 +484,9 @@ def __init__(
         self.root.protocol("WM_DELETE_WINDOW", self.on_close)
 
         # Set window icon
-        self.icon_image = ImageTk.PhotoImage(file="assets/logo.png")
+        self.icon_image = ImageTk.PhotoImage(
+            file=str(Path(__file__).parent.parent) + "/assets/logo.png"
```

Comment:
It looks like `.parent.parent` should be just `.parent`. Otherwise, I get this error:
```
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\ProgramData\\miniconda3\\Lib\\site-packages\\assets\\logo.png'
```